docs/proposals/20240807-in-place-updates.md

```diff
@@ -489,7 +492,7 @@ The `vsphere-vm-memory-update` extension does not support any or the required ch
 }
 ```
 
-Given the there are still changes not covered, KCP continues with the next update extension, making the same request to the `kcp-version-upgrade/CanUpdateMachine` endpoint of the `kcp-version-upgrade` extension:
+Given that there are still changes not covered, KCP continue with the next update extension, making the same request to the `kcp-version-upgrade/CanUpdateMachine` endpoint of the `kcp-version-upgrade` extension:
```

Comment:
i think this is not correct
there is no `CanUpdateMachine` endpoint with the new design